src/dotnet/commands/dotnet-list/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Tools.List.ProjectToProjectReferences;
+
+namespace Microsoft.DotNet.Tools.List
+{
+    public class ListCommand : DispatchCommand
+    {
+        protected override string HelpText => @".NET List Command
```

Comment:
Add them to a LocalizableStrings file. Don't worry about the xlf files for the time being. That said, these strings have missed the localization train this time around. Update the PR to follow the common pattern and we'll need to figure out how to handle these new strings moving forward.